backend/coreapp/compilers.py

```diff
@@ -538,6 +538,12 @@ def available_platforms() -> List[Platform]:
     cc='${COMPILER_DIR}/bin/ee-gcc -c $COMPILER_FLAGS "$INPUT" -o "$OUTPUT"',
 )
 
+EE_GCC29_991111B_R4 = GCCPS2Compiler(
+    id="ee-gcc2.9-991111b-r4",
+    platform=PS2,
+    cc='${COMPILER_DIR}/bin/ee-gcc -c $COMPILER_FLAGS "$INPUT" -o "$OUTPUT"',
```

Comment:
this is a Windows executable `ee-gcc.exe`, so would need to be invoked with wine or wibo.
```
$file ee-gcc.exe
ee-gcc.exe: PE32 executable (console) Intel 80386, for MS Windows
```